docs/getting_started_guide/Installation_Options/ISO/Installing_SIMP_From_An_ISO.rst

```diff
@@ -3,91 +3,128 @@
 Installing SIMP from an ISO
 ===========================
 
-The benefits of using a SIMP ISO are:
+SIMP can be installed from a bootable ISO, which provides many advantages:
 
-* Suitable for enclave or offline environments
-* It is the easiest way to get started and ensure that all files are present
-* Your SIMP load will have a disk partitioning scheme compatible with most
-  security guides
-* Your system will start in :term:`FIPS` mode
-* Your disks can be encrypted
+* Provides a ready-to-go OS, Puppet server, and SIMP installation
+* Suitable for use in network-isolated enclaves/offline environments
+* Ensures OS is configured with compliance-relevant install-time options:
 
-  * Please pay attention to the caveats in the :ref:`ig-disk-encryption`
-    section
+  * The :ref:`ig-disk-partitioning` scheme is compatible with most security
+    guides
+  * The OS will boot in :term:`FIPS` mode
+  * Disks will be encrypted
 
-Obtaining the ISO
------------------
+    * (Please note the important caveats in the :ref:`ig-disk-encryption`
+      section)
+
+
+Obtaining a SIMP ISO file
+-------------------------
 
-The SIMP ISO can be downloaded from the `official SIMP ISO Share`_.
+You can obtain a SIMP installation ISO using one of the following methods:
 
-Alternatively, you can compile your own ISO by following the documentation in
-:ref:`gsg-building_a_simp_iso`.
+#. Downloading an ISO image file
 
-Installation
-------------
+   * Official releases are available at https://download.simp-project.com/simp/ISO/
 
-The ISO will install on any system that supports the underlying operating system.
+#. *[Advanced]* :ref:`gsg-building_a_simp_iso` for yourself
 
-.. WARNING::
+   * ISO can be customized to your preferences.
+   * This is the only way to obtain a SIMP ISO that installs a licensed
+     commercial OS, such as Red Hat Enterprise Linux (:term:`RHEL`).
 
-   There are default passwords present on the system that should be changed
-   prior to deploying the system.
 
-   **Please make sure that you change these passwords!**
+Installing the OS
+-----------------
+
+A SIMP ISO will install its OS + SIMP on any host that supports the underlying
+operating system.
 
 Install as follows:
 
-#. Boot the system using the SIMP ISO
+#. Boot the system using the SIMP ISO.
+
+   The ISO will load into a screen of boot options, with instructions:
 
-#. Press *Enter* to run the standard SIMP install, or choose from the
-   customized options list.
+   .. image:: ../../../images/screenshots/simp_boot_options.png
```

Comment:
This is the BIOS boot screen.  The UEFI boot screens (sequence) are different.  Were you planning adding UEFI screens in this PR?

Reply:
@lnemsick-simp No, I wasn't—but it's a good idea.